branding.py

```diff
@@ -0,0 +1,36 @@
+BRAND_CONSOLE_URL = "https://xcp-ng.org"
```

Comment:
I would prefer if this file `.github/branding.py` and be moved to `branding.py` by the github workflow.

Placing it directly is bound to produce errors in the .spec files because the file needs to be overwritten